kong-dbless.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  namespace: kong
+  name: kong-control-plane
+  labels:
+    app: kong-control-plane
+spec:
+  selector:
+    matchLabels:
+      app: kong-control-plane
```

Comment:
This name doesn't sound right. Kong in DB-less mode is merely a proxy with a bunch of config. Should this be names something simpler?

Reply:
I wasn't sure what to call it figured it'd get debated on the PR ;)

Is just `kong` sufficient I wonder?